README.md

```diff
@@ -220,6 +220,144 @@ C:\Users\bion\Desktop>dumpbin /IMPORTS .\example.exe | findstr msvcp
     msvcp140d_oss.dll
 ```
 
+# How To Run The Tests From The Developer Command Prompt For VS
+
+1. Follow steps 1-9 of
+[How To Build With A Native Tools Command Prompt](#how-to-build-with-a-native-tools-command-prompt).
+2. Invoke `git submodule update --init llvm-project`
+3. Invoke `cmake -G Ninja -DCMAKE_TOOLCHAIN_FILE={where your vcpkg clone is located}\scripts\buildsystems\vcpkg.cmake
+-DENABLE_TESTS=TRUE -S . -B {wherever you want binaries}`. This differs from above only in `-DENABLE_TESTS=TRUE`.
+4. If you have already followed the steps from
+[How To Build With A Native Tools Command Prompt](#how-to-build-with-a-native-tools-command-prompt), and have not
+changed the value of {wherever you want binaries} in step four, then there is no need to rebuild to run the tests.
+Otherwise, invoke `ninja -C {wherever you want binaries}` to build the project.
+
+In addition to following the above steps you must also have [Python][] 3.8 or newer, and have LLVM's `bin` directory on
+the PATH.
+
+## Running All The Tests
+
+After configuring and building the project, running `ctest` from the build output directory will run all the tests.
+CTest will only display the standard error output of tests that failed. In order to get more details from CTest's
+`lit` invocations, run the tests with `ctest -V`.
+
+## Running A Subset Of The Tests
+
+`${PROJECT_BINARY_DIR}\tests\llvm-lit\llvm-lit.py` can be invoked on a subdirectory of a testsuite and will execute all
+the tests under that subdirectory. This can mean executing the entirety of a single testsuite, running all tests under
+a category in libcxx, or running a single test in `std` and `tr1`.
+
+## Examples
+
+```
+:: This command configures the project with tests enabled. It assumes you are using the vcpkg submodule and have
+:: already installed boost. It also assumes you have inited and updated the llvm-project submodule.
+C:\STL\build>cmake -GNinja -DCMAKE_CXX_COMPILER=cl -DCMAKE_TOOLCHAIN_FILE=..\vcpkg\scripts\buildsystems\vcpkg.cmake^
+ -DENABLE_TESTS=TRUE ..

```

Comment:
I think it would be much less confusing to put the space before the caret:
```suggestion
C:\STL\build>cmake -GNinja -DCMAKE_CXX_COMPILER=cl -DCMAKE_TOOLCHAIN_FILE=..\vcpkg\scripts\buildsystems\vcpkg.cmake ^
-DENABLE_TESTS=TRUE ..
```